float64 distance
float64 heading
string layer_name
time last_seen